Harvest.Web/Controllers/ProjectController.cs

```diff
@@ -147,6 +152,15 @@ public async Task<ActionResult> Get(int projectId)
             return Ok(project);
         }
 
+        [HttpGet]
+        [Authorize(Policy = AccessCodes.FieldManagerAccess)]
+        public async Task<ActionResult> GetFields()
+        {
+            var fields = await _dbContext.Fields.ToListAsync();
```

Comment:
Should this be limited to fields marked IsActive and/or whose projects are active?

Reply:
That is a good point. I will change that

Reply:
Oh, also I think it should be authorized with `AccessCodes.WorkerAccess`.

Reply:
@wongband It looks like new fields default to being marked IsActive, and there's no logic that changes it to false, so be sure to include `field.Project.IsActive && field.Project.Status == Project.Statuses.Active` in the filter.

Reply:
@sprucely I noticed that all of the fields' Project field is null. Is it suppose to be like this?

Reply:
@wongband It'll work in the query.  The field is null when you get the result from the database because it won't pull relations by default.  But it'll work from the query itself since it just gets transformed to SQL.  If you really do need the project full data you can do a `.Include()` but I don't think you need that for this case.